service.toml

```diff
@@ -1,6 +1,11 @@
 name = "azfile"
 
 [namespace.storage]
+features = ["virtual_dir"]
```

Comment:
`dir` is natively supported in azfile. We can implement `Direr` with the public APIs instead of simulate it.

Reply:
Got it!